src/main/java/io/vertx/rabbitmq/RabbitMQClient.java

```diff
@@ -43,6 +46,15 @@ static RabbitMQClient create(Vertx vertx, RabbitMQOptions config) {
     return new RabbitMQClientImpl(vertx, config);
   }
 
+
+  /**
+   * Set a callback to be called whenever a new connection is established.
+   * This callback must be idempotent - it will be called each time a connection is established, which may be multiple times against the same instance.
+   * @param connectionEstablishedCallback 
+   */
+  @GenIgnore
+  void addConnectionEstablishedCallback(Runnable connectionEstablishedCallback);
```

Comment:
can you instead use `Handler<Void>` ?

Reply:
Had to make it Handler<RabbitMQClient> anyway, because of race conditions in the robust consumer.

Reply:
ok